packages/core/template/project/DeveloperGuide.md

```diff
@@ -0,0 +1,285 @@
+---
+  layout: default.md
+  title: "Developer Guide"
+  pageNav: 3
+---
+
+# ProjectEx Developer Guide
+
+<!-- * Table of Contents -->
+<page-nav-print />
+
+--------------------------------------------------------------------------------------------------------------------
+
+## **Acknowledgements**
+
+_{ list here sources of all reused/adapted ideas, code, documentation, and third-party libraries -- include links to the original source as well }_
+
+--------------------------------------------------------------------------------------------------------------------
+
+## **Setting up, getting started**
+
+Refer to the guide [_Setting up and getting started_](SettingUp.md).
+
+--------------------------------------------------------------------------------------------------------------------
+
+## **Design**
+
+<box type="tip">
+This section describes the architecture of your app, explaining how the main components work and interact with each other. Using architecture diagrams is recommended.
+</box>
+
+### Architecture
+
+The ***Architecture Diagram*** given below explains the high-level design of the ProjectEx App.
+
+<img src="images/johndoe.png" width="200px">
+<box type="info" seamless>
+
+Architecture Diagram of your project created using [`puml`](https://markbind.org/userGuide/components/imagesAndDiagrams.html#diagrams) feature.
```

Comment:
Earlier photo is particularly confusing because it is Not using PUML... Can also consider using PUML instead of an image to show how it is incorporated? 

But I am OK so long as it is clear that this is just a demo, and the image above is not intended to be a PUML diagram